### DISCUSSION TITLE
is there easy and fast way to Transform React components into HTML email templates?

### DISCUSSION THREAD

**Discussion by Greatz08** [resend/react-email#1662] [General]
Right now we have to do with cli code so is there any faster way? or any thoughts on creating addon/extension to simplify this?

**Comment by BelenLuo-tech** [comment]
Yes, similar to exporting multiple html based on multiple languages, there is no way to support

**Comment by gabrielmfern** [comment]
There is the [`render`](https://react.email/docs/utilities/render). It won't turn into an HTML email template per se, as React Email is not meant to do that specifically, but it will render into pure HTML.

If you might be under some misconception of how you should use React Email let me clarify. React Email is meant to give you the components, and the method of converting React into HTML, such that you can use React with props. This would be the method you use to give input into the email, to then render into HTML at the moment of sending the email. This is the best way in terms of DX, and in terms of performance there is not much of an impact. 

Here's an example using [Resend + Next.JS](https://github.com/resend/react-email/blob/canary/examples/resend/src/pages/api/send.ts).
